test/functional/feature_evm_mempool.py

```diff
@@ -215,10 +214,20 @@ def mempool_tx_limit(self):
 
         # Mint a block
         self.nodes[0].generate(1)
-        self.blockHash = self.nodes[0].getblockhash(self.nodes[0].getblockcount())
+        height_checkpoint = self.nodes[0].getblockcount()
+
+        # Check that now we can send again.
+        self.nodes[0].evmtx(self.ethAddress, nonce, 21, 21001, self.toAddress, 1)
+        self.nodes[0].generate(1)
         block_txs = self.nodes[0].getblock(
             self.nodes[0].getblockhash(self.nodes[0].getblockcount())
         )["tx"]
+        assert_equal(len(block_txs), 2)
+
+        self.rollback_to(height_checkpoint)
```

Comment:
Dont quite get the point of the rollback though. Doesnt the original test already cover this? We try to send another evm tx from the same sender subsequently after spamming and minting the block.

Reply:
You're right, I missed it. Since it was after the EVM accounting fees.